thrift-confignode/src/main/thrift/confignode.thrift

```diff
@@ -53,12 +53,37 @@ struct DataPartitionInfo {
     2: required map<i32, list<i32>> dataRegionIDsMap
 }
 
+struct DeviceGroupHashInfo {
+    1: required i32 deviceGroupCount
+    2: required string hashClass
+}
+
+struct DataNodeRegisterReq {
+    1: required rpc.EndPoint endPoint
+}
+
+struct DataNodeRegisterResp {
```

Comment:
Maybe we can also return `DeviceGroupHashInfo` in this struct to save one extra rpc?

Reply:
How about fix it in next pr? There must exist other parameters that needs to be returned in DataNode registration.